compiler/tests/testSuite.bal

```diff
@@ -27,6 +28,12 @@ function testCompileVPO(string path, string kind) returns io:Error? {
     }
 }
 
+function verifyErrorFileName(err:Any err) {
+    err:Detail detail = <err:Detail>err.detail();
+    test:assertTrue(detail.location is err:Location, "error without location");
+    test:assertNotEquals((<err:Location>detail.location).filename.length(), 0, "error with an empty filename");
```

Comment:
We need to check that the filename is right, not just that it is non-empty.

Reply:
fixed